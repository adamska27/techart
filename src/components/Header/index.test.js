import React from 'react';
import { Link } from 'react-router-dom';

import { shallow } from 'enzyme';

import Component, { ContainerHeader } from './index';

// useless test just fro test
describe('Header', () => {
  it('should render without throwing error', () => {
    expect(
      shallow(<Component />).contains(
        <Link to="/">
          <li>Accueil</li>
        </Link>
      )
    ).toBe(true);
  });
});
