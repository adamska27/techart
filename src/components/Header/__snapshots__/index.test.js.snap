// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header match the Header snapshot 1`] = `
<header
  className="sc-bxivhb gwefrW"
>
  <nav
    className="sc-ifAKCX eYlISq"
  >
    <div
      className="sc-bwzfXH gzCqZe"
      onClick={[Function]}
    >
      <img
        src="btnNavMobile.svg"
      />
    </div>
    <a
      href="/"
      onClick={[Function]}
    >
      <div
        className="sc-EHOje gefODq"
      >
        Logo
      </div>
    </a>
    <ul
      className="sc-bZQynM jYZVnb"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <li>
          Accueil
        </li>
      </a>
      <a
        href="/games"
        onClick={[Function]}
      >
        <li>
          Jeux
        </li>
      </a>
      <a
        href="/members"
        onClick={[Function]}
      >
        <li>
          membres
        </li>
      </a>
      <a
        href="/mygames"
        onClick={[Function]}
      >
        <li>
          mes jeux
        </li>
      </a>
    </ul>
    <ul
      className="sc-bdVaJa cqVyzt"
    >
      <a
        href="/signup"
        onClick={[Function]}
      >
        <li>
          Inscription
        </li>
      </a>
      <a
        href="/signin"
        onClick={[Function]}
      >
        <li>
          Connexion
        </li>
      </a>
    </ul>
  </nav>
</header>
`;
